activerecord/test/cases/adapters/postgresql/connection_test.rb

```diff
@@ -90,7 +90,7 @@ def test_reset_with_transaction
     end
 
     def test_tables_logs_name
-      @connection.tables('hello')
+      ActiveSupport::Deprecation.silence { @connection.tables('hello') }
```

Comment:
Should be the same behavior on `data_sources`. Can we rewrite that test to no longer use `tables`?